V2/tezex/src/pages/removeLiquidity/index.tsx

```diff
@@ -0,0 +1,97 @@
+import { FC, useState, useEffect } from "react";
+
+import { BigNumber } from "bignumber.js";
+import {
+	TokenInput,
+} from "../../components/ui/elements/inputs";
+import { TokenAmountOutput } from "../../components/ui/elements/Labels";
+import { useWallet } from "../../hooks/wallet";
+import { useNetwork } from "../../hooks/network";
+import {
+	lqtOutput,
+	removeLiquidity,
+} from "../../functions/liquidityBaking";
+import { Transact } from "../../components/ui/elements/Buttons";
+
+import { TokenKind } from "../../types/general";
+
+export interface IRemoveLiquidity {
+	children: null;
+}
+export const RemoveLiquidity: FC = (props) => {
```

Comment:
the entire function is commented out, is it needed?

Reply:
no